### PR TITLE
Add support for zombie options

### PR DESCRIPTION
**Pull request by berliner**
This is a follow up of https://github.com/minkphp/MinkZombieDriver/pull/154.

This PR allows to specify zombie options as an array argument to the server constructor.
I have applied more or less the same logic as for the threshold argument.
Tests have been updated/added too.
I didn't update the README, but instead would issue a PR against the docs repo once this PR here has been approved.
The associated PR for Behat/MinkExtension is https://github.com/Behat/MinkExtension/pull/284

### CODE REVIEW

File: src/NodeJS/Server.php [hunk 1]
```diff
@@ -274,6 +282,37 @@ public function getThreshold()
...
+     * Setter options value
+     *
+     * @param array $options Options array
+     *
+     * @throws \InvalidArgumentException Invalid path is invalid.
+     * @throws \LogicException When server is already running.
+     */
+    public function setOptions($options)
+    {
+        if (!is_array($options)) {
```
Comment (stof) [inline src/NodeJS/Server.php]: use a typehint instead

File: src/NodeJS/Server.php [hunk 2]
```diff
@@ -46,6 +46,11 @@
     protected $threshold;
 
     /**
+     * @var array
+     */
+    protected $options;
```
Comment (stof) [inline src/NodeJS/Server.php]: please use private visibility for the new property (as it has both a getter and a setter which are public, there is no benefit making this accessible through inheritance, and protected stuff makes it harder to handle backward compatibility)

Comment (berliner) [inline src/NodeJS/Server.php]: @stof I tried to follow what's already there and all the other properties are protected while having setters and getters.

File: src/NodeJS/Server.php (hunk 1 above)
Comment (berliner) [inline src/NodeJS/Server.php]: done

File: src/NodeJS/Server.php (hunk 2 above)
Comment (aik099) [inline src/NodeJS/Server.php]: @berliner , please make this change. I guess that's last thing before this PR is merged.

Comment (berliner) [inline src/NodeJS/Server.php]: @aik099 Done, though I still feel that this doesn't belong into this PR.

**Comment by aik099** [comment]
Merging, thanks @berliner .

**Pull request merged**